src/components/BestProductCard.jsx

```diff
@@ -0,0 +1,27 @@
+import likes from "../assets/icons/likes.svg";
+import basic from "../assets/images/basic.svg";
+
+function ProductCard({ name, images, price, favoriteCount }) {
+  const changePrice = price.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
```

Comment:
💊 제안
JS 내장 메소드인 toLocaleString 함수를 통해 더 쉽게 구현이 가능합니다~

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString

```suggestion
  const changePrice = price.toLocaleString();
```